### ISSUE TITLE
Recipe #32: Simple Collection 

### ISSUE THREAD

**Issue by glenrobson** [IIIF/trc#123]
## Links

 * Original Issue: https://github.com/IIIF/cookbook-recipes/issues/32
 * Pull Request: https://github.com/IIIF/cookbook-recipes/pull/371
 * Preview: https://preview.iiif.io/cookbook/0032-collection/recipe/0032-collection/ 

## Background and Summary

This is a recipe that was missing showing the basics of how to build a very simple collection. We have more complex recipes looking at collections including Newspapers and bound volumes but were missing the basic example. The collection in the recipe has two artworks which are individual manifests and are linked in a Collection. 

## Voting and changes

We welcome comments on the recipe and as well as voting +1, confused face or -1 feel free to add comments to this issue. If this issue is approved then the author will take account of the comments before we merge the branch in to the master cookbook branch.

If the recipe is rejected by the TRC then we will make the changes requested and resubmit it to a future TRC meeting. If you feel that your comments are substantial enough that the recipe should be looked at again by the TRC after the changes have been made please vote -1 (thumbs down). A confused face is treated as abstaining. 

Changes to the recipe will only be made after the TRC voting process has concluded.

**Comment by markpbaggett** [comment]
@robcast brought up a good point in TRC that it would be nice to link to the Presentation API specification so that it's easy to understand where the rules about behaviors and inheritance are defined.  Maybe we could just link directly to [the behavior section](https://iiif.io/api/presentation/3.0/#behavior) in the Implementation Notes section?

**Comment by zimeon** [comment]
I agree with the comment above that there should be a link to the [the behavior section](https://iiif.io/api/presentation/3.0/#behavior) in Prezim, but I think it should in the somewhat tricky comment on inheritance. On closer inspection, the literal `behavior` is already linked but one doesn't notice it because of the code font. Perhaps a a link could be added on "behaviors" later in that sentence, either additionally or instead, which will show up better?
> Collections may have [`behavior`](https://iiif.io/api/presentation/3.0/#behavior) defined in order to influence user experience when viewing the resource. Manifests within a Collection DO NOT inherit [behaviors](https://iiif.io/api/presentation/3.0/#behavior) (<-- ADD LINK) from their referencing Collections; however referenced Collections will.

**Comment by glenrobson** [comment]
### Issue 123 (Recipe #32: Simple Collection )
  +1: 18 [akrishnan15 cubap dlpierce hadro ioanrichards irv julsraemy ksclarke ktamaral markpatton markpbaggett mikeapp nfreire peteredwards regisrob rentonsa robcast zimeon]
   0: 0 []
  -1: 0 []
  Not TRC: 0 []
  Ineligible: 1 [mathewjordan]

### Result: 18 / 18 = 1.00
Super majority is in favor, issue is approved
